new/PRS13274AmIya.xml

```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13274AmIya" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ʾAmda ʾIyasus</title>
```

Comment:
Etymologically it is ʿAmda. Do we standardize proper names although their spellings in the ms are non-etymological?

Reply:
> Etymologically it is ʿAmda. Do we standardize proper names although their spellings in the ms are non-etymological?

I would suggest to standardize the proper names.

Reply:
@thea-m or @DariaElagina Waht is the BM policy for Person records: name shall be standardized according to Dillmann ?

Reply:
There is no policy. If you decide to standardise, I think it would be good to state it.

Reply:
To state it in the record?

Reply:
I think that would make sense

Reply:
I will change the person records and add < note. However, in the manuscrupt record, I follow the wrting of the manuscript, correct?